File: lab008/cliente/models/departamento.dart
class Departamento {
  final int? id;
  final String nombre;
  final String descripcion;
  final String telefono;
  final String fax;

  Departamento(this.id, this.nombre, this.descripcion, this.telefono, this.fax);

  factory Departamento.fromJson(Map<String, dynamic> json) {
    return Departamento(
      json['idDpto'] as int?,
      json['nombre'] as String,
      json['descripcion'] as String,
      json['telefono'] as String,
      json['fax'] as String,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'idDpto': id,
      'nombre': nombre,
      'descripcion': descripcion,
      'telefono': telefono,
      'fax': fax,
    };
  }
}

File: lab009/servidor/models/Departamento.java
package com.ocho.miguel.models;

public class Departamento {

    private int idDpto;
    private String nombre;
    private String descripcion;
    private String telefono;
    private String fax;

    public Departamento(int idDpto, String nombre, String descripcion, String telefono, String fax) {
        this.idDpto = idDpto;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.telefono = telefono;
        this.fax = fax;
    }

    public int getIdDpto() {
        return idDpto;
    }

    public void setIdDpto(int idDpto) {
        this.idDpto = idDpto;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getFax() {
        return fax;
    }

    public void setFax(String fax) {
        this.fax = fax;
    }
}
